__mocks__/@inrupt/solid-client.js

```diff
@@ -0,0 +1,18 @@
+import { vi } from 'vitest'
+import { mockSolidDatasetFrom } from '@inrupt/solid-client';
+
+export * from '@inrupt/solid-client';
+
+const exampleUrl = 'https://example.com'
+
+export const getPodUrlAll = vi.fn(() => Promise.resolve([]));
+export const saveAclFor = vi.fn((url) => Promise.resolve(
+  {
+    ...mockSolidDatasetFrom(url),
+    internal_accessTo: exampleUrl
+  }));
+export const saveSolidDatasetAt = vi.fn((url) => Promise.resolve(mockSolidDatasetFrom(url)));
+export const getSolidDataset = vi.fn((url) => Promise.resolve(mockSolidDatasetFrom(url)));
+export const setAgentResourceAccess = vi.fn();
```

Comment:
These `setResourceAccess` functions aren't networked, but they require a specific format that's only generated by correctly accessing the network, and that isn't provided by `mockSolidDatasetFrom`. If that format is not provided, they error.

Mocking these here because we generally aren't interested in the output of these functions themselves; we just pass the output around. If you are interested in the output, mock the return value.